geometry.pyi

```diff
@@ -118,6 +118,7 @@ class Line(Sequence[float]):
     def move_ip(self, x: float, y: float) -> None: ...
     @overload
     def move_ip(self, move_by: Coordinate) -> None: ...
+    def at(self, t: float) -> Tuple[float, float]: ...
```

Comment:
Given that this is basically a lerp, wouldn't something like `weight` or `fac` be better suited as "t"?